### PR TITLE
chore(operator): release notes for 1.8.0-1.8.4

### PR DESCRIPTION
**Pull request by mrampton**

### CODE REVIEW

**Review by dogonthehorizon — APPROVED** [review]

File: content/en/continuous-deployment/release-notes/rn-armory-operator/armory-operator-v1-8-1.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Armory scans the codebase as we develop and release software. For information about CVE scans for this release, contact your Armory account representative.
+
+## Known issues
+
+No known issues.
+
+## Highlighted updates
+
+Operator now supports disabling the Admission Controller.
```
Comment (aimeeu) [inline content/en/continuous-deployment/release-notes/rn-armory-operator/armory-operator-v1-8-1.md:21]: I'm assuming someone who uses the Armory Operator would know what "disabling the Admission Controller" means

File: content/en/continuous-deployment/release-notes/rn-armory-operator/armory-operator-v1-8-3.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Armory scans the codebase as we develop and release software. For information about CVE scans for this release, contact your Armory account representative.
+
+## Known issues
+
+Armory Operator may fail to start in some EKS clusters. Please upgrade to Armory Operator v1.8.4.  
+
+## Highlighted updates
+
+* Support for basic installation as an alternative RBAC model
```
Comment (aimeeu) [inline content/en/continuous-deployment/release-notes/rn-armory-operator/armory-operator-v1-8-3.md]: Again, assuming a user would know what "Support for basic installation as an alternative RBAC model" means. I don't recall anything in the Operator docs about this, but I could be wrong

**Review by aimeeu — APPROVED** [review]
2 minor comments but nothing to hold up merging

File: content/en/continuous-deployment/release-notes/rn-armory-operator/armory-operator-v1-8-1.md (hunk 1 above)
Comment (dogonthehorizon) [reply content/en/continuous-deployment/release-notes/rn-armory-operator/armory-operator-v1-8-1.md:21]: They should know about Admission Controllers, yeah

File: content/en/continuous-deployment/release-notes/rn-armory-operator/armory-operator-v1-8-3.md (hunk 2 above)
Comment (dogonthehorizon) [inline content/en/continuous-deployment/release-notes/rn-armory-operator/armory-operator-v1-8-3.md]:
```suggestion
* Helm chart support for basic-mode installation pattern
```

Comment (dogonthehorizon) [reply content/en/continuous-deployment/release-notes/rn-armory-operator/armory-operator-v1-8-3.md]: This is enabling the basic (as opposed to cluster) level installation method that we call out in our docs *from the Helm chart* that the ATIPO team consumes. I've suggested a rewording here to make that more clear.

**Review by dogonthehorizon — APPROVED** [review]

**Pull request merged**